build.sbt

```diff
@@ -391,7 +391,7 @@ lazy val client = libraryCrossProject("client")
       nettyCodecHttp % Test
     )
   )
-  .dependsOn(core, server, testing % "test->test", theDsl % "test->compile")
+  .dependsOn(core, server % "test", testing % "test->test", theDsl % "test->compile")
```

Comment:
This one is 💯 on me in https://github.com/http4s/http4s/pull/5563, sorry everyone 😳

Reply:
while I'm here 😉 

```suggestion
  .dependsOn(core, server % Test, testing % "test->test", theDsl % "test->compile")
```